<launch>
    <arg name="gui" default="true"/>
    <arg name="verbose" default="true"/>
    <arg name="world" default="empty.world"/>

    <include file="robot_in_gazebo.launch">
        <arg name="world" value="$(arg world)"/>
        <arg name="use_victor" value="false"/>
        <arg name="use_val" value="true"/>
        <arg name="robot_name" value="hdt_michigan"/>
        <arg name="verbose" value="$(arg verbose)"/>
        <arg name="gui" value="$(arg gui)"/>
        <arg name="pause" value="true"/>
        <arg name="debug_gazebo" value="false"/>
        <arg name="debug_moveit" value="false"/>
    </include>
</launch>
